tokio-console/Cargo.toml

```
@@ -33,13 +33,13 @@ clap = { version = "3", features = ["cargo", "derive", "env"] }
 tokio = { version = "1", features = ["full", "rt-multi-thread"] }
 tonic = { version = "0.9", features = ["transport"] }
 futures = "0.3"
-tui = { version = "0.16.0", default-features = false, features = ["crossterm"] }
+tui = { package = "ratatui", version = "0.20.1", default-features = false, features = ["crossterm"] }
```

We'd prefer to fully switch to `ratatui`, rather than using it under the name of `tui`.

```suggestion
ratatui = { version = "0.20.1", default-features = false, features = ["crossterm"] }
```

---

Done